docker/test-images/zipkin-elasticsearch7/Dockerfile

```diff
@@ -27,34 +27,35 @@ FROM scratch as scratch
 COPY build-bin/docker/docker-healthcheck /docker-bin/
 COPY docker/test-images/zipkin-elasticsearch7/start-elasticsearch /docker-bin/
 COPY docker/test-images/zipkin-elasticsearch7/config/ /config/
+COPY docker/test-images/zipkin-elasticsearch7/NOTICE.txt /
 
 FROM ghcr.io/openzipkin/java:${java_version} as install
 
 WORKDIR /install
 
-# Use latest 7.x version from https://www.elastic.co/downloads/past-releases#elasticsearch
+# Use latest 7.x version from https://www.elastic.co/downloads/past-releases#elasticsearch-no-jdk
```

Comment:
@xeraa why is there no 8.x here?

Reply:
looks like we're to delete the jdk directory, which is fine https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html#jvm-version

Reply:
Yeah, we don't build those any more. It's recommended to stick to the default JDK we provide, but you can delete the shipped JDK folder (optional) and link to any JDK you like (with or without deleting the default JDK)